src/sagas/utils.js

```diff
@@ -162,7 +162,10 @@ export function* delayInMsSteps (count = 20, msMultiplier = 2000) {
 
 export function* fetchPermits ({ entityType, id }) {
   const permissions = yield callExternalAction(`get${entityType}Permissions`, Api[`get${entityType}Permissions`], { payload: { id } })
-  return getUserPermits(Api.permissionsToInternal({ permissions: permissions.permission }))
+  if (permissions && Array.isArray(permissions.permission)) {
+    return getUserPermits(Api.permissionsToInternal({ permissions: permissions.permission }))
```

Comment:
`Api.permissionsToInternal` is called in 2 places - the other place already had the check so the problem is not that rare. This suggest that it would be better to have the fix in one common place. 
How about simply `toInternal ({ permissions = [] }` ?

Reply:
@rszwajko It's a great idea so I added the default value as you suggested.
IMO we don't want to "work" with the response without check its content so I kept the if statement.
Thanks :)

Reply:
I would remove the checks now and rely only on default value for error handling but it depends what flow we want here:
current error handling (no action in case of error) does not clear existing permissions which might be a desired side effect..
We can as well have both the explicit check(main path) and default value (as safety net)